eZ/Publish/API/Repository/Tests/RoleServiceTest.php

```diff
@@ -1708,7 +1745,7 @@ public function testGetRoleAssignments()
 
         /* END: Use Case */
 
-        $this->assertEquals(1, count($roleAssignments));
+        $this->assertEquals(2, count($roleAssignments));
```

Comment:
This is a consequence of starting to read and use the role assignment ID. Before, multiple assignments of the same role to the same user/group with different subtree limitations, for example, would be concatenated into one assignment object with an array of multiple limitation values.

But the assignment ID refers to one and only one limitation value, so after the change we don't concatenate assignments anymore, which leads to the count increase. More assignment objects, the same number of limitations.